compiler/noirc_frontend/src/parser/parser/types.rs

```diff
@@ -14,7 +14,7 @@ use crate::token::{Keyword, Token, TokenKind};
 use chumsky::prelude::*;
 use noirc_errors::Span;
 
-pub(super) fn parse_type<'a>() -> impl NoirParser<UnresolvedType> + 'a {
+pub fn parse_type<'a>() -> impl NoirParser<UnresolvedType> + 'a {
```

Comment:
Can this be
```suggestion
pub(crate) fn parse_type<'a>() -> impl NoirParser<UnresolvedType> + 'a {
```
?

Reply:
It needs to be `pub` due to being publicly exported along with some other parser functions.